libraries/consul_service.rb

```diff
@@ -55,6 +55,7 @@ def default_environment
         {
           'GOMAXPROCS' => [node['cpu']['total'], 2].max.to_s,
           'PATH' => '/usr/local/bin:/usr/bin:/bin',
+          'SHELL' => node['consul']['service_shell'],
```

Comment:
@legal90, question for you.

At this point, do we want to make this separate value from `node['consul']['service_shell']`? If we do, the user could still have no valid shell and we could define a separate shell for when we're running in systemd. The value could either be hardcoded or another attribute.